packages/integration_test/lib/integration_test.dart

```diff
@@ -447,3 +457,29 @@ class _GarbageCollectionInfo {
   final int oldCount;
   final int newCount;
 }
+
+/// Connect to the given uri and return a new [VmService] instance.
+///
+/// Copied from vm_service_io so that we can pass a custom [HttpClient] for
+/// testing. Currently, the WebSocket API reuses an HttpClient that
+/// is created before the test can change the HttpOverrides.
+Future<vm.VmService> _vmServiceConnectUri(
```

Comment:
use `//` since these docs would need rewording before making this public

Reply:
Done